updatecfg.go

```diff
@@ -168,6 +169,10 @@ func (cmd *updateConfigCommand) run() error {
 		return err
 	}
 
+	if reflect.ValueOf(ls).IsNil() {
+		return errors.New("the releases from helm list is empty")
```

Comment:
I think If there's an error here, it means we cannot find the release named xxxx, but not helm list is empty, right?

Reply:
Right, updated.